test/Dialect/Verif/prepare-for-formal.mlir

```diff
@@ -0,0 +1,45 @@
+// RUN: circt-opt --prepare-for-formal %s | FileCheck %s
+module {
+  // CHECK-LABEL: @FlattenWires
+  hw.module @FlattenWires(in %clock : !seq.clock, in %reset : i1, in %in : i1, out out : i32) {
+
+    //CHECK:  %false = hw.constant false
```

Comment:
Hyper-nit: Please change this to `// CHECK`, for consistency with the `CHECK-LABEL` above and in other tests.

A quick grep shows there's 412 instances of that (`//CHECK` presently vs 27745 with `// CHECK` .